Monika After Story/game/zz_calendar.rpy

```diff
@@ -2055,6 +2180,8 @@ screen calendar_overlay():
     #     xalign 0.305
     #     yalign 0.4
     #
+    image "mod_assets/calendar/calendar_button_shadow.png" xpos 351 ypos 251
+
     if store.mas_calendar.enabled:
```

Comment:
Looks and works good!
Mind if I ask you to add one bit (yes, it has nothing to do with the fonts, but still)? It was bothering me for a long time. When you open the calendar, the button that you use to open it is still active (you can hear the hovering sound). I think we could add a check if we're showing the calendar or not. And if we are, we make the button inactive.
```suggestion
    if (
        store.mas_calendar.enabled
        and renpy.get_screen("mas_calendar_screen") is None
    ):
```

Reply:
Ah, sure, I've been meaning to fix it anyway, but completely forgot about it. Thanks!

_(This PR is already out of scope, so...)_